README.md

```diff
@@ -1,7 +1,7 @@
 # :package_fullname
 
 [![Latest Version on Packagist](https://img.shields.io/packagist/v/tightenco/:package_name.svg?style=flat-square)](https://packagist.org/packages/tightenco/:package_name)
-[![GitHub Tests Action Status](https://img.shields.io/github/workflow/status/tighten/:package_name/run-tests?label=tests)](https://github.com/tighten/:package_name/actions?query=workflow%3Arun-tests+branch%3Amain)
+[![GitHub Tests Action Status](https://img.shields.io/github/actions/workflow/status/tighten/:package_name/.github/workflows/tests.yml?branch=main&label=tests)](https://github.com/tighten/:package_name/actions?query=workflow%3Arun-tests+branch%3Amain)
```

Comment:
Dang, I saw the note from them about changing this but didn't realize we also had to add the `.github/workflows` folder. Good catch!